en/docs/getting-started/quick-start-guide.md

```diff
@@ -239,101 +239,110 @@ Let's look at how you can use the CI/CD command line tool for APIs (API Controll
-                `--apim` : API Manager endpoint for the environments  
-     
-             -   Optional :      
-                 `--admin` : Admin endpoint for the environment 
-                 `--token` : Token endpoint for the environment 
-                 `--registration` : Registration endpoint for the environment                   
-                 `--publisher` : Publisher endpoint for the environment 
-                 `--devportal` : DevPortal endpoint for the environment
-   
+        - Required :     
+        `--environment` or `-e` : Name of the environment to be added   
+        `--apim` : API Manager endpoint for the environments  
 
-     
-  
+        - Optional :      
+            `--admin` : Admin endpoint for the environment 
+            `--token` : Token endpoint for the environment 
+            `--registration` : Registration endpoint for the environment                   
+            `--publisher` : Publisher endpoint for the environment 
+            `--devportal` : DevPortal endpoint for the environment
+   
 
      On successfully executing this command, you should see the following message.
      
      `Successfully added environment 'dev'`
 
 ### Step 1 - Create an API
-       
-1. Initialize an API project by providing a name for the project. 
 
-     Let's use the command below to create an API named `PetstoreAPI`.  This creates a folder named `PetstoreAPI` in your current directory.
+1. Initialize an API project by providing a name for the project.
+
+     Let's use the command below to create an API named `PetstoreAPI`. This creates a folder named `PetstoreAPI` in your current directory.
 
        ```bash
-       ./apictl init PetstoreAPI --oas https://petstore.swagger.io/v2/swagger.json
-       ``` 
+       ./apictl init PetstoreAPI --oas https://raw.githubusercontent.com/OAI/OpenAPI-Specification/master/examples/v3.0/petstore.json
+       ```
+
+     On successfully executing this command, you should see the following message.
+
+     ``` text
+     Project initialized
+     Open README file to learn more
+     ```   
     
     !!! note
 
-        Use the following command to view the various options related to initializing a project. 
+        Use the following command to view the various options related to initializing a project.
 
         ```bash
         ./apictl init --help
         ```
 
-2. Open the `api.yaml` file. 
+2. Open the `api.yaml` file.
 
      Open and explore the `PetstoreAPI` folder with an IDE (e.g., VSCode). Navigate to the `Meta-information` directory and open the `api.yaml` file.
 
     !!! tip
 
         Alternatively, You can use a text editor to open this file as well.
    
-     Change the values of the attributes `status` and `productionUrl` as shown below and save the file. 
+     Change the value of the attribute `status` as shown below and save the file.
        
-     `status: PUBLISHED`  
-     `productionUrl: http://petstore.swagger.io/v2`  
```

Comment:
Does the API invocation work without the `productionUrl` being specified?

Reply:
This section mentions what the user has to change. The default values are, so I left it at that and it worked.
```
productionUrl: http://localhost:8080
sandboxUrl: http://localhost:8081
``

Reply:
Updated in https://github.com/wso2/docs-apim/pull/1587/commits/6a8d54053ad94e900da141df0ff7fa9a91e275cb